src/homework4/decorator.py

```diff
@@ -0,0 +1,18 @@
+"""
+Создайте декоратор, который хранит в себе результаты вызовов функций
+и время получения результата за время запуска программы.
+"""
+
+import time
+
+
+def log_results(lst):
```

Comment:
It's a good idea but I want just to decorate function, not create other variables etc.